tests/test_vpc.rb

```diff
@@ -21,12 +21,13 @@
 # TestVpc - Minitest for Vpc node utility class
 class TestVpc < CiscoTestCase
   def setup
+    skip("Test not supported on #{node.product_id}") if platform != :nexus
```

Comment:
FWIW I'm also adding the skip_unless_supported as part of #250 - so whichever of us commits first will win here ;)